utilities/upgrade.py

```
@@ -0,0 +1,69 @@
+        "MTV_VERSION": mtv_version,
+        "MTV_SOURCE": mtv_source.upper(),
+        "IMAGE_INDEX": image_index,
+        "CLUSTER_USERNAME": py_config["cluster_username"],
+        "CLUSTER_PASSWORD": py_config["cluster_password"],
+        "CLUSTER_API_URL": py_config["cluster_host"],
+    })
+
+    LOGGER.info(f"Running MTV upgrade: {script_path} (version={mtv_version}, source={mtv_source}, index={image_index})")
+
+    try:
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)